### DISCUSSION TITLE
Windows 11 Do Not Disturb Actions

### DISCUSSION THREAD

**Discussion by Joecascio2000** [Codectory/AutoActions#126] [Ideas]
I would love to be able to enable and disable windows 11 do not disturb mode based on application. The delivered automatic rules are so limiting.

**Comment by Codectory** [comment]
My time is also limited, we are building a house  and I just don’t find the
time to continue AutoActions right now, although this would be a really
great feature

Joecascio2000 ***@***.***> schrieb am Sa. 24. Sept. 2022 um
06:50:
